.meteor/versions

```diff
@@ -151,5 +151,3 @@ url@1.2.0
 webapp@1.6.2
 webapp-hashing@1.0.9
 zimme:active-route@2.3.2
-mizzao:timesync@0.5.0	
```

Comment:
I think we need these two packages, but I'm not sure why meteor seems to remove them every time on startup.

Reply:
I think this explains it https://github.com/mizzao/meteor-timesync/issues/49

Reply:
Great find!!